composition/CLAUDE.md

```
@@ -0,0 +1,269 @@
+
+```typescript
+{ fieldName: "", selectionSet: "id", disableEntityResolver?: true }
+```
+
+Stored in `configurationData.keys` array — one entry per @key directive.
+
+## Entity Caching — Current State and Known Limitations
+
+### Directive Hierarchy
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)